### ISSUE TITLE
Update second level domains for .ck according the current registrar [general utility]

### ISSUE THREAD

**Issue by derwyddon** [publicsuffix/list#1942]
According to the registrar (from IANA page https://www.iana.org/domains/root/db/ck.html)  https://www.vodafone.co.ck/domain-admin-policy the allowed SLD for .ck are seven (and six are publicly available, except gov.ck):
The .ck TLD has the current entry on the PSL:

// ck : https://en.wikipedia.org/wiki/.ck
*.ck
!www.ck

The proposal is:
// ck : https://en.wikipedia.org/wiki/.ck
ck
biz.ck
co.ck
edu.ck
gen.ck
net.ck
org.ck
gov.ck

or even updating the sources to: https://www.vodafone.co.ck/domain-admin-policy

According to https://tld-list.com/tld/ck there exist an additional info.ck SLD available, but it does not appear on vodafone page.

**Comment by dnsguru** [comment]
Excellent, please contact the nic to add the respective _PSL txt records to the zone files and request they submit a pull request.

**Comment by simon-friedberger** [comment]
@dnsguru More general question: Do you know if IANA requires some kind of point of contact that we could forward such requests to?

**Comment by dnsguru** [comment]
Yes , Caveat: Good News, Bad News...

**Good News:** [Database of Top Level Domains](https://www.iana.org/domains/root/db)  email listed per TLD

**Bad News:**  Because it is a publicly listed email address, the TLD admins get thundering herds of unsolicited junk email, so it is atypical to ever hear a response to volunteer messages - this is where my 15 years of standing in front of ccTLDs in different events like Tech Day during ICANN, Registry Operators Workshop, CENTr, APTLD, LACTLD, etc as outreach and myrelationships in the ccTLD and gTLD registry community has advanced the project.

**Comment by mozfreddyb** [comment]
Is there a list of private contacts we could start sharing internally?

**Comment by dnsguru** [comment]
Best to refer to #1439 and move along.  We in all cases want the registry to be intentional about the manner in which their listing is set in the PSL, which means some form of direct interaction with the TLD in question.

> Is there a list of private contacts we could start sharing internally?

Not really.  I (or some other volunteer) typically have to hunt down the current contact on a case-by-case basis.

The people at IANA can sometimes be helpful, but there is a spectrum of levels of engagement that ccTLD admins have even with them which ranges in frequency between never in touch except to roll keys or update NS delegations or alter contact

Crucially, a list like this would likely not be valuable to our cause.  These are pop-up needs, and staff turnover at registries and privacy laws confound this.  Additionally, WHEN these are needed, it it typically when the back-end provider has changed, so any old contact would not be with the present admin.

In most cases I am working downward from the CEO connections I have built over decades, or my presence at an ICANN meeting in the ccNSO or TechDay allows me an enhanced "Where's Waldo?" opportunity.

What we have trended towards is #1439 - so that the reporting party self-volunteers to do the follow-up work with the affected registry, or engages the registry to file a PR, if needed.

In all cases, there can be some ccTLDs that have not enumerated their sub-allocations in the PSL, and some that have done so.  It is really about ensuring that however configured, it is intentional.   After 15 years of constantly standing in front of that community educating on what the PSL is and advocating/encouraging ccTLD admins for PSL review of their entries, I can say that it needs attention from time to time as it pops up.

Additional dialog with the IANA where information has been provided to them about where to direct ccTLD admins for their PSL updates, I can say that there is a very reasonable assumption that entries are intentional at this point.
